apollo-ios/Sources/Apollo/GraphQLError.swift

```diff
@@ -86,3 +86,14 @@ extension GraphQLError: LocalizedError {
     return self.description
   }
 }
+
+extension GraphQLError {
+  public func asDictionary() -> [String: Any] {
```

Comment:
Could we make this an `internal` function instead? Unless there is value in letting users turn their `GraphQLError` instances into JSON, but they get that already through `GraphQLResult.asDictionary()`?

Reply:
Yea good point, probably not a need to expose this publicly necessarily